Create community.yml

justwheel opened the pull request:
This file aims to provide a new area of contribution for a specific type of community collaborator. Until recently, there was not an area where contributors of this type could easily help with the Project, but the CommOps subgroup aims to help target this kind of collaborator, and more help is needed with these efforts. This is a beginning list of areas that contributors could check out to become involved.

There's not any new badges for the CommOps team yet, but this is subject to change, so I recycled badges from other areas for images.

Commit:
Create community.yml

This file aims to provide a new area of contribution for a specific type of community collaborator. Until recently, there was not an area where contributors of this type could easily help with the Project, but the CommOps subgroup aims to help target this kind of collaborator, and more help is needed with these efforts. This is a beginning list of areas that contributors could check out to become involved.

There's not any new badges for the CommOps team yet, but this is subject to change.

ralphbean commented:
Looks pretty good to me.  Do you want to include a link for it in the base file?  https://github.com/fedora-infra/asknot-ng/blob/develop/questions/fedora.yml

Can you get the site to build locally?  How does it look when you build it and render it in a browser?

Commit:
Add "Community" section to fedora.yml

Adds a section on the main page to add the community.yml file.

justwheel commented:
@ralphbean: Just added that to the base file. Haven't had a chance to test it yet, but will do so shortly, in another hour or two. Don't have my environment quite ready on this machine! Will reply once I've tested and ensured my changes are up to par.

Commit:
Use consistent wording (Community => Comm. Ops)

Changes wording in base file to be consistent with the original wording in the `community.yml` file. Community => Community Operations.

ralphbean commented:
I'll give it a run, @jflory7.  Thanks!
